packages/neuron-wallet/src/models/chain/transaction.ts

```diff
@@ -72,6 +77,7 @@ export default class Transaction {
   public nftInfo?: NFTInfo
 
   public signatures: Signatures = {}
+  public assetAccountType?: AssetAccountType | string = '' // Default to ''
```

Comment:
How about only use `AssetAccountType`? It can be `AssetAccountType` or undefined.